tests/integration/service_test.py

```diff
@@ -902,7 +902,7 @@ def test_log_drive_empty_default_jsonfile(self):
         self.assertFalse(log_config['Config'])
 
     def test_log_drive_none(self):
-        service = self.create_service('web', log_driver='none')
+        service = self.create_service('web', logging={'driver': 'none'})
```

Comment:
nevermind, I guess at this point the config has been normalized to the v2 format.